handler/handler.go

```diff
@@ -29,7 +30,11 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, cfg *config.Config
 	}
 	log.Info(ctx, "event handler called", logData)
 
-	contentPublished, err := h.ZebedeeCli.GetPublishedData(ctx, event.URL)
+	// Strip off data.json
+	uri := strings.TrimSuffix(event.URL, "/data.json")
```

Comment:
I don't think this will work as expected if the URL contains a query string. I don't know enough about this task to know if that is a realistic scenario but it's something to consider.

Reply:
Furthermore, I don't think this is the right place to do this. I think we should be stripping this off in zebedee before sending the events. The event should be sending over the URIs of pages to be indexed not the uris of the data.json.